scripts/generate_spherical_expansion_ref_data.py

```diff
@@ -96,7 +81,8 @@ def dump_reference_json():
                                 # x = get_soap_vectors(hypers, frames)
                                 sph_expn = SphericalExpansion(**hypers)
                                 expansions = sph_expn.transform(frames)
-                                x = expansions.get_feature_matrix()
+                                x = expansions.get_dense_feature_matrix(sph_expn)
+                                x[np.abs(x) < 1e-300] = 0.
```

Comment:
this is a very silly bug... 
we return some numbers that are 1e-326 which is a bit outside of the typical range of doubles. And the json library transforms them into some python object that the c++ json reader does not understand. So I put these small numbers to 0 to avoid the problem.